src/sources/docker.rs

```diff
@@ -580,8 +592,21 @@ impl EventStreamBuilder {
                 }
             })
             .take_while(|v| future::ready(v.is_ok()))
-            .filter_map(|v| future::ready(v.transpose()))
```

Comment:
Why `transponse()` changed to `unwrap()`?

Reply:
The `take_while` before it ends the stream if `v` is Err, so we will always have `Ok` here. Right @MOZGIII?

Reply:
We need to get rid of the wrapping to pass the event to the `line_agg_adapter`. It doesn't really matter where we do it (could be done inside the adapter). It's ok because of the previous line - no `Err` items will ever be at the `unwrap()`. This is one of the things I'd refactor to be less cognitively complex.

Reply:
Indeed, `unwrap()` will always successful because we have filter on previous line. Thanks.